core-ap/processor/src/test/kotlin/org/enginehub/piston/RegressionTest.kt

```diff
@@ -157,4 +157,31 @@ class RegressionTest {
         }
     }
 
+    @Test
+    @DisplayName("issue #18, regarding sub-command aliases")
+    fun issue18SubCommandAliases() {
+        withRegressionCommands { _, manager ->
+            manager.register("hello") { cmd ->
+                val sub = manager.newCommand("world")
+                        .action { SUB_ACTION }
+                        .aliases(setOf("there"))
+                        .description(TextComponent.of("Sub-command"))
+                        .build()
+                cmd.run {
+                    action { 0x43110 }
```

Comment:
You can drop this, there's a default NULL_ACTION. It's used above because we want to check when it is called, but that's not important here.